.browserslistrc

```diff
@@ -0,0 +1 @@
+defaults
```

Comment:
Add this file to .gitignore and run `git rm --cached .browserlistrc` to untrack this file

Reply:
you don't fix it